### DISCUSSION TITLE
equests.exceptions.SSLError: HTTPSConnectionPool(host='api.openai.com', port=443): Max retries exceeded with url: /v1/chat/completions (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:997)')))

### DISCUSSION THREAD

**Discussion by EdisonChenZX** [acheong08/ChatGPT#1203] [Q&A]
I report this error when using  "requests.exceptions.SSLError: HTTPSConnectionPool(host='api.openai.com', port=443): Max retries exceeded with url: /v1/chat/completions (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:997)')))"

**Comment by CoolPlayLin** [comment]
Could you please show your source code?
